src/misc/lv_anim.h

```diff
@@ -511,7 +511,7 @@ static inline bool lv_anim_custom_delete(lv_anim_t * a, lv_anim_custom_exec_cb_t
  * @param exec_cb   a function pointer which is animating 'var', or NULL to return first matching 'var'
  * @return          pointer to the animation.
  */
-static inline lv_anim_t * lv_anim_custom_get(lv_anim_t * a, lv_anim_custom_exec_cb_t exec_cb)
+static inline lv_anim_t * lv_anim_custom_get(const lv_anim_t * a, lv_anim_custom_exec_cb_t exec_cb)
```

Comment:
```suggestion
static inline lv_anim_t * lv_anim_custom_get(lv_anim_t * a, lv_anim_custom_exec_cb_t exec_cb)
```

It needs to reverted for the MicroPython binding.